lib/ansible/modules/cloud/amazon/rds_instance_facts.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
```

Comment:
I take your point, but as this is a WIP, I'd rather just rebase on the results of the other PR once that's included. Just ignore the facts modules for the time being - I expect the facts PR to be merged long before this one.